schema/bom-1.4-strict-SNAPSHOT.schema.json

```diff
@@ -1074,6 +1082,278 @@
           "description": "The value of the property."
         }
       }
+    },
+    "advisory": {
+      "type": "object",
+      "title": "Advisory",
+      "description": "Location of additional advisory information.",
+      "required": ["url"],
+      "additionalProperties": false,
+      "properties": {
+        "title": {
+          "type": "string"
+        },
+        "url": {
+          "type": "string",
+          "format": "iri-reference"
+        }
+      }
+    },
+    "cwe": {
+      "type": "integer",
+      "minimum": 1,
+      "title": "CWE",
+      "description": "Integer representation of a Common Weaknesses Enumerations (CWE). For example 399 (of https://cwe.mitre.org/data/definitions/399.html)"
+    },
+    "severity": {
+      "type": "string",
+      "title": "Severity",
+      "description": "Textual representation of the severity of the vulnerability adopted by the risk analysis method. If an other risk analysis method is used other than whats defined in scoreSourceType, the user is expected to translate appropriately to match with an element value below.",
+      "enum": [
+        "critical",
+        "high",
+        "medium",
+        "low",
+        "none",
+        "unknown"
+      ]
+    },
+    "scoreMethod": {
+      "type": "string",
+      "title": "method",
+      "description": "Specifies the risk scoring methodology/standard used.",
+      "enum": [
+        "CVSSv2",
+        "CVSSv3",
+        "CVSSv31",
+        "OWASP",
+        "Other"
+      ]
+    },
+    "impactAnalysisState": {
+      "type": "string",
+      "title": "Impact Analysis State",
+      "description": "Textual representation of the state of the impact analysis.",
+      "enum": [
+        "exploitable",
```

Comment:
Should we handle direct and indirect exploitability?

Should we handle confirmed and unconfirmed exploitability?

Reply:
Description now states exploitable = direct or indirect exploitability.  No interest in supporting confirmed and unconfirmed at this time.